### ISSUE TITLE
Question on the object detection phase

### ISSUE THREAD

**Issue by yangshao** [GriffinLiang/vrd-dsr#6]
As in the detection phase, both the bounding box and bounding box labels are required, so are you using fast-rcnn's pre-trained model or you use fast-rcnn to re-train on the vrd and vg dataset? Besides, as ranking loss require sample negative predicates, how do you do the sampling?

**Comment by GriffinLiang** [comment]
For detection phase, it is better to use the pre-trained model from Pascal or COCO dataset. 
For the proposed ranking loss, we use all the negative predicates without sampling.

**Comment by yangshao** [comment]
@GriffinLiang If using the pre-trained model, then there will be some labels which does not occur on the target object vocabulary, how do you handle this?

**Comment by GriffinLiang** [comment]
Discard the last layer from the pre-trained model, initialize it randomly and fine-tune the whole model on the target dataset.

**Comment by yangshao** [comment]
@GriffinLiang  I'm still confused by the data split for the relation detection task. during the training phase, the gold bounding box and gold labels are used? as the training data is both used for object detection and relation detection.

**Comment by GriffinLiang** [comment]
Yes. The ground-truth boxes and labels are used for both object detection and relation detection.
